### DISCUSSION TITLE
Include 'getting to a minimum viable cd'

### DISCUSSION THREAD

**Discussion by rlabercrombie** [Minimum-CD/cd-manifesto#199] [Q&A] [answered]
Given the latest presentation at devnexus (great job by the way!) there were a few questions about 'how to get there' with tough architectures; in this example they mentioned having to have a many year long process of transitioning from VMs to x/y/z prior to being able to achieve (or it sounded like even start) minimal cd.  

Would there be any interest in creating a new section or adding to existing sections for use-cases fitting the bill of 'getting to a minimum viable cd system: best practices (or examples... whichever would fit best)?  

As an example of a use-case: You have a monolith with zero testing coverage.   Separating out a small chunk of code - ideally on an appropriate domain/business seam - into its own project, adding some unit (or other) tests and getting a basic pipeline working as a way of an iterative process? Then you could have parallel work on improving that pipeline and pulling off another small chunk of code.  

Or is this out of scope?

**Comment by bdfinst** [comment, marked as answer]
I think this falls squarely in scope for community experience reports or tips. We are incrementally adding these as we go. I'm currently working on an expanded section for each of the core behaviors to explain why each is important. Adding something on the stranger pattern or even "how to I CD this mudball?" would be useful indeed.

**Reply by bdfinst** [reply]
Issue opened here: https://github.com/Minimum-CD/cd-manifesto/issues/200
